enginetest/engine_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	sqle "github.com/dolthub/go-mysql-server"
 	"github.com/dolthub/go-mysql-server/enginetest"
 	"github.com/dolthub/go-mysql-server/enginetest/queries"
+	setup2 "github.com/dolthub/go-mysql-server/enginetest/scriptgen/setup"
```

Comment:
Fix this alias (probably introduced because you have a variable named setup somewhere)